lib/pages/discover.dart

```diff
@@ -70,7 +78,10 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
                     header: MaterialClassicHeader(),
                     controller: _refreshController,
                     onRefresh: scan,
-                    child: ListView.builder(
+                    child: _isListEmpty ?
```

Comment:
You can use `deviceList.isEmpty` here directly? I think it's fulfilling the same purpose.

Reply:
@sanket143 yes I tried that initially, but what was happening was that both the refresher and the Text appeared on the screen at the same time. The text stayed there since the beginning and didn't let refresher to complete loading.